examples/ranker/config.xml

```diff
@@ -0,0 +1,7 @@
+<View>
+  <Style>
+    .htx-ranker-column { background: cornflowerblue; }
+    .htx-ranker-item { background: lightgoldenrodyellow; }
+  </Style>
+  <Ranker name="ranker" value="$items" mode="rank" title="Search Results|Relevant Results"/>
```

Comment:
There are two titles for one-column mode, I think it's better to leave only one here.